pages/common/qc.md

```diff
@@ -0,0 +1,24 @@
+# qc
+
+> CLI snippet manager to execute command snippets stored in notes in [QOwnNotes](https://www.qownnotes.org/).
+> More information: <https://www.qownnotes.org/getting-started/command-line-snippet-manager.html>.
+
+- Configure the snippet manager, e.g. to set the security token from QOwnNotes:
+
+`qc configure`
+
+- Search for snippets stored in your `Commands.md`:
```

Comment:
Maybe we can update this description to match with your docs?

```suggestion
- Search and print command snippets:
```

Alternatively, we could reword it to:

```suggestion
- Search and print command snippets stored in the `Commands.md` file:
```

We generally avoid using terms like "your", etc in the descriptions. Feel free to reword my suggestions.

Reply:
Is the filename `Commands.md` hard coded? Or does it allow specifying different file names too? In that case, can you update the example for the same?

Reply:
`Commands.md` is just the default. The user can change that.